tools/c7n_azure/c7n_azure/function.py

```diff
@@ -39,22 +39,12 @@ def main(input):
 
     event = None
 
-    if type(input) is HttpRequest:
-        event = input.get_json()
+    if type(input) is QueueMessage:
+        event = [input.get_json()]
```

Comment:
Should we change AzureEventGridMode::run function instead of making an array here?

Reply:
Andy and I discussed this before and decided keeping as array will make it more extensible in the future.

Reply:
Can we rename event to events in that case?

Reply:
sure